closure/compiler/closure_js_library.bzl

```diff
@@ -68,6 +68,25 @@ def _impl(ctx):
     args += ["--expect_failure"]
   if ctx.attr.internal_expect_warnings:
     args += ["--expect_warnings"]
+
+  files = [ctx.outputs.provided, ctx.outputs.stderr]
+  if ctx.attr.style and len(ctx.files.srcs) > 0:
+    files.append(ctx.outputs.jsstyle)
+    style_args = [ctx.executable._clangformat.path,
+                  ctx.outputs.jsstyle.path]
+    style_inputs = [ctx.executable._clangformat]
+    for direct_src in ctx.files.srcs:
+      style_args += [direct_src.path]
+      style_inputs.append(direct_src)
+    ctx.action(
```

Comment:
When we do the `closure_js_library` integration in a follow-up pull request, do you think it would be better if we launched clang format as a concurrent subprocess of JsChecker? The advantage is that with a significant amount of kludgery, we might be able to integrate it into JsChecker's error system. Therefore clang format errors could be turned off via suppress rather than a new attribute. How feasible does that sound, based on your understanding of clang format so far?